pkg/filters/args.go

```diff
@@ -12,20 +12,22 @@ import (
 )
 
 type ArgFilter struct {
-	filters map[events.ID]map[string]Filter
+	filters map[events.ID]map[string]Filter[*StringFilter]
 	enabled bool
 }
 
+var _ utils.Cloner[*ArgFilter] = &ArgFilter{}
```

Comment:
This is for type assertion right? Maybe add a comment so it won't be accidentally removed.